src/Simulation/AutoSubstitution/Microsoft.Quantum.AutoSubstitution.csproj

```diff
@@ -20,11 +20,11 @@
   
   <ItemGroup>
     <PackageReference Include="Microsoft.Quantum.Compiler" Version="0.17.2106147878-beta" />
+    <PackageReference Include="Microsoft.Quantum.CSharpGeneration" Version="0.17.2106147878-beta" IsQscReference="true" />
```

Comment:
If we remove `<CSharpGeneration>false</CSharpGeneration>`, can we remove this `PackageReference`?